assets/companies/components/CompaniesApp.jsx

```diff
@@ -75,30 +52,11 @@ CompaniesApp.propTypes = {
     dispatch: PropTypes.func,
 };
 
-const mapStateToProps = (state) => ({
-    companies: state.companies.map((id) => state.companiesById[id]),
-    companyToEdit: state.companyToEdit,
-    activeCompanyId: state.activeCompanyId,
-    isLoading: state.isLoading,
-    activeQuery: state.activeQuery,
-    totalCompanies: state.totalCompanies,
-    companyOptions: state.companyOptions,
-    companiesById: state.companiesById,
-    companyUsers: state.companyUsers,
-    errors: state.errors,
-});
 
 const mapDispatchToProps = (dispatch) => ({
```

Comment:
fyi this could be a dict and then it wraps all it's values into dispatch, so this could be:
`const mapDispatchToProps = {
    newCompany,
    fetchCompanies,
    setQuery,
};
```